### PR TITLE
enable-all linters on golangci-lint

### PR DESCRIPTION
**Pull request by andrey1s**
can you integrate lint with https://golangci.com/ ?

### CODE REVIEW

File: codegen/templates/templates_test.go [hunk 1]
```diff
@@ -9,58 +9,106 @@ import (
...
-	require.Equal(t, "relatedUrls", ToGoPrivate("RelatedUrls"))
-	require.Equal(t, "iTicket", ToGoPrivate("ITicket"))
-
-	require.Equal(t, "rangeArg", ToGoPrivate("Range"))
-
-	require.Equal(t, "a", ToGoPrivate("A"))
-	require.Equal(t, "id", ToGoPrivate("ID"))
-	require.Equal(t, "id", ToGoPrivate("id"))
-	require.Equal(t, "", ToGoPrivate(""))
+	cases := map[string]struct {
```
Comment (vektah) [inline codegen/templates/templates_test.go]: :'( This is a big step backwards in readability. Lets kill this rule.

File: codegen/testserver/tracer_test.go [hunk 2]
```diff
@@ -115,45 +115,45 @@ func TestTracer(t *testing.T) {
 
 		configurableTracer := &configurableTracer{
 			StartOperationParsingCallback: func(ctx context.Context) context.Context {
-				return context.WithValue(ctx, "StartOperationParsing", true)
+				return context.WithValue(ctx, ctxStartOperationParsing, true)
```
Comment (vektah) [inline codegen/testserver/tracer_test.go]: :-1:

File: graphql/bool.go [hunk 3]
```diff
@@ -6,6 +6,8 @@ import (
 	"strings"
 )
 
+const nameTrue = "true"
```
Comment (vektah) [inline graphql/bool.go]: :-1:

File: handler/playground.go [hunk 4]
```diff
@@ -44,7 +44,7 @@ var page = template.Must(template.New("graphiql").Parse(`<!DOCTYPE html>
 `))
 
 func Playground(title string, endpoint string) http.HandlerFunc {
-	return func(w http.ResponseWriter, r *http.Request) {
+	return func(w http.ResponseWriter, _ *http.Request) {
```
Comment (vektah) [inline handler/playground.go]: :-1:

File: .golangci.yml [hunk 5]
```diff
@@ -1,3 +1,22 @@
+run:
+  tests: true
+  skip-dirs:
+    - bin
+    - vendor
+    - var
+    - tmp
+    - example/*
```
Comment (vektah) [inline .golangci.yml]: no coverage of example?

File: .golangci.yml [hunk 6]
```diff
@@ -1,3 +1,22 @@
+run:
+  tests: true
+  skip-dirs:
+    - bin
+    - vendor
```
Comment (vektah) [inline .golangci.yml]: we shouldn't have a vendor dir?  have these just been copied from somewhere?

File: .golangci.yml [hunk 7]
```diff
@@ -1,3 +1,22 @@
...
+    - tmp
+    - example/*
+
 linters-settings:
   errcheck:
     ignore: fmt:.*,[rR]ead|[wW]rite|[cC]lose,io:Copy
+
+linters:
+  enable-all: true
+  disable:
```
Comment (vektah) [inline .golangci.yml]: a long list of enabled linters is probably better than removing from a changing target imo.

**Comment by andrey1s** [comment]
revert `golint` and `goconst` linters

**Pull request merged**